jsonnet/components/prometheus.libsonnet

```diff
@@ -167,6 +167,12 @@ function(params)
           resourceNames: ['nonroot'],
           verbs: ['use'],
         },
+        {
+          // Access to the Prometheus / Thanos HTTP API through kube-rbac-proxy
```

Comment:
Eventually there's no reason to grant permissions on prometheuses/api to the Prometheus SA. But I get that the origin e2e tests depend on this assumption so fine with it for now.

Reply:
exactly. this is to please the origin test only.

Reply:
Can you update the comment to make it clear?

Reply:
yes